scripts/run_integration_tests.sh

```diff
@@ -30,29 +30,53 @@ function cleanup () {
 
 RUNDIR=$(dirname ${0})
 SKIP_JS_BUILD=
+TEST_AGAINST_FREQUENCY_ROCOCO=0
+TEST_AGAINST_LOCAL_RELAY=0
+
 trap 'cleanup EXIT' EXIT
 trap 'cleanup TERM' TERM
 trap 'cleanup INT' INT
 
-while getopts "s" OPTNAME
+while getopts "srl" OPTNAME
 do
     case "${OPTNAME}" in
-        "s") SKIP_JS_BUILD=1
+        "s")
+            SKIP_JS_BUILD=1
+        ;;
+        "r")
+            TEST_AGAINST_FREQUENCY_ROCOCO=1
+        ;;
+        "l")
+            TEST_AGAINST_LOCAL_RELAY=1
```

Comment:
For completeness the shell script should return an error if both of these cli options are passed as they are mutually exclusive

Reply:
Seems like a custom enum implementation would be overkill & non-standard. I lean toward the KISS principle 😄 -- stick with 0/1 and an appropriately named variable.

Reply:
(However, on further thought, a single CLI option `-c` (`c` for "chain") that takes an argument of "local" or "rococo" might be simpler, and eliminate the need for the mutually-exclusive option check. Since there only two strings to check for, a hand-rolled enum implementation seems like overkill (though it's a pretty cool pattern), but the overall principal is similar.)

Reply:
I like this idea